File: ZombieFIST/ZombieFist/ngApp/lists/planList.viewModel.ts
namespace zombieApp.lists {

    // Typescript class that holds our profile info
    export class planListViewModel {
        // not properties that are saved, but are used as 
        // a way to keep track of an item being added.
        newItem: string;
        newItemError: boolean;

        // property for tracking if in edit or display state
        isEdit: boolean;


        id: number;
        name: string;
        items: Array<planListItemViewModel>;
    }
}

File: ZombieFIST/ZombieFist/Models/ViewModels/PlanListViewModel.cs
using System.Collections.Generic;

namespace ZombieFist.Models.ViewModels
{
    public class PlanListViewModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public IList<PlanListItemViewModel> Items { get; set; }
    }
}
